testsuite/MDAnalysisTests/coordinates/test_openmm.py

```diff
@@ -0,0 +1,241 @@
+
+import MDAnalysis as mda
+
+mm = pytest.importorskip("simtk.openmm")
+unit = pytest.importorskip("simtk.unit")
+app = pytest.importorskip("simtk.openmm.app")
+
+class TestOpenMMBasicSimulationReader():
+    @pytest.fixture
+    def omm_sim_uni(self):
+        system = mm.System()
+        topology = app.Topology()
+        chain = topology.addChain("CHAIN")
+        hydrogen = app.element.Element.getByAtomicNumber(1)
+        residue = topology.addResidue("RES", chain)
+        for i in range(5):
+            system.addParticle(1.0)
+            topology.addAtom(hydrogen.symbol, hydrogen, residue)
+        positions = np.ones((5,3)) * unit.angstrom
+        integrator = mm.LangevinIntegrator(273 * unit.kelvin,
+            1.0 / unit.picoseconds, 2.0 * unit.femtoseconds)
+        simulation = app.Simulation(topology, system, integrator)
+        simulation.context.setPositions(positions)
+
+        return mda.Universe(simulation)
+
+
+    def test_dimensions(self, omm_sim_uni):
+        assert_almost_equal(
+            omm_sim_uni.trajectory.ts.dimensions,
+            np.array([20., 20., 20., 90., 90., 90.]),
+            3,
+            "OpenMMBasicSimulationReader failed to get unitcell dimensions " +
+            "from OpenMM Simulation Object",
+        )
+
+    def test_coordinates(self, omm_sim_uni):
+        up = omm_sim_uni.atoms.positions
+        reference = np.ones((5,3))
+        assert_almost_equal(up, reference, decimal=3)
+
+    def test_basic_topology(self, omm_sim_uni):
+        assert omm_sim_uni.atoms.n_atoms == 5
+        assert omm_sim_uni.residues.n_residues == 1
+        assert omm_sim_uni.residues.resnames[0] == "RES"
+        assert omm_sim_uni.segments.n_segments == 1
+        assert omm_sim_uni.segments.segids[0] == 0
+        assert len(omm_sim_uni.bonds.indices) == 0
+
+
+
+
+class TestOpenMMPDBFileReader(_SingleFrameReader):
+    __test__ = True
+
+    def setUp(self):
+        self.universe = mda.Universe(app.PDBFile(RefAdKSmall.filename))
+        self.ref = mda.Universe(RefAdKSmall.filename)
+        self.prec = 3
+
+    def test_dimensions(self):
+        assert_almost_equal(
+            # Angles seem to parse differently when openmm reads the pdb file
+            self.universe.trajectory.ts.dimensions[0:3],
+            self.ref.trajectory.ts.dimensions[0:3],
+            self.prec,
+            "OpenMMPDBFileReader failed to get unitcell dimensions " +
+            "from OpenMMPDBFile",
+        )
+
+    def test_coordinates(self):
+        up = self.universe.atoms.positions
+        rp = self.ref.atoms.positions
+        assert_almost_equal(up, rp, decimal=3)
+
+
+class TestOpenMMModellerReader(_SingleFrameReader):
+    __test__ = True
+
+    def setUp(self):
+        pdb_obj = app.PDBFile(RefAdKSmall.filename)
+        modeller = app.Modeller(pdb_obj.topology, pdb_obj.positions)
+        self.universe = mda.Universe(modeller)
+        self.ref = mda.Universe(RefAdKSmall.filename)
+        self.prec = 3
```

Comment:
You only use this once, it is easier to understand if you just use 3 in the test below.

Reply:
It seems like self.prec gets called somewhere in the base tests like in test_distance. I can hardcode the precision in the specific openmm tests, but it still seems like setting self.prec seems important for other tests?

Reply:
Ah, if it ain't broke, let's not fix it.